src/test/java/org/jenkinsci/plugins/secure_requester_whitelist/CasCTest.java

```diff
@@ -0,0 +1,48 @@
+package org.jenkinsci.plugins.secure_requester_whitelist;
+
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import io.jenkins.plugins.casc.snakeyaml.Yaml;
+import jenkins.model.Jenkins;
+import org.hamcrest.CoreMatchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+
+import java.io.StringReader;
+import java.util.Map;
+
+import static org.hamcrest.collection.IsMapContaining.hasEntry;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class CasCTest {
+
+    @Rule
+    public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
+
+    @Test
+    @Issue("JENKINS-57616")
+    @ConfiguredWithCode("CasCTest.yml")
+    public void configureWhitelist() throws Exception {
```

Comment:
You can extend from a test class yo make this cleaner (I think). See https://github.com/jenkinsci/ant-plugin/pull/36 as en example.

Reply:
I'm not sure what you mean, what makes the ant example cleaner?

Reply:
Basically you save the rule, the @test, the @configureWithCode.